.idea/compiler.xml

```diff
@@ -60,8 +81,10 @@
       <module name="fdb-record-layer-spatial_integrationTest" target="1.8" />
       <module name="fdb-record-layer-spatial_main" target="1.8" />
       <module name="fdb-record-layer-spatial_test" target="1.8" />
+      <module name="fdb-record-layer.main" target="11" />
+      <module name="fdb-record-layer.test" target="11" />
       <module name="fdb-record-layer_main" target="1.8" />
       <module name="fdb-record-layer_test" target="1.8" />
     </bytecodeTargetLevel>
   </component>
-</project>
+</project>
```

Comment:
I assume this change wasn't intentionally checked in?

Reply:
Doh! No, it isn't.  I need to try to understand what is up with this file.  In my environment, it won't compile without it, so I typically have to stash it when I commit, then restore it after.  Anyway, it is removed now